README.markdown

```diff
@@ -26,12 +30,12 @@ This style guide is based on the raywenderlich.com Style Guide (https://github.c
 
 ## Naming
 
-Use descriptive names with camel case for classes, methods, variables, etc. Class names and constants in module scope should be capitalized, while method names and variables should start with a lower case letter.
+Use descriptive names with camel case for classes, methods, variables, etc. Type names and protocols should be capitalized, while method names and variables should start with a lower case letter.
 
 **Preferred:**
 
 ```swift
-let MaximumWidgetCount = 100
+private let maximumWidgetCount = 100
```

Comment:
Please remove **private** from here. This mixes rules for privates we use the _ notation.